includes/Checker/Checks/Plugin_Readme_Check.php

```diff
@@ -457,7 +477,11 @@ private function get_wordpress_stable_version() {
 				if ( isset( $body['offers'] ) && ! empty( $body['offers'] ) ) {
 					$latest_release = reset( $body['offers'] );
 
-					$version = $latest_release['new_bundled'];
+					$version = $latest_release['current'];
+
+					if ( preg_match( '#^\d.\d#', $version, $matches ) ) {
+						$version = $matches[0];
+					}
```

Comment:
I assume you changed this because `new_bundled` seems to be outdated in the API?

Let's revert this change here and instead make sure the API is fixed :-)

Reply:
But I don't understand current is current version of WordPress, right?

Reply:
But current is `6.5.5`. We only need `6.5.`

We should not need to do any regex whatsoever to get from `6.5.5` to `6.5`.

The `new_bundled` field already provides the right value for us.

Reply:
But it's not giving. I think the fastest way is making a regex as it's in the PR

Reply:
But we're not in a rush, no need to use the "fastest" way. Also, arguably, simply fixing the API is faster.

Definitely an opponent of doing unnecessary regex checks like this.